libraries/recipe.rb

```diff
@@ -63,4 +63,46 @@ def self.keys(node)
     order << ''
     order
   end
+
+  # Return Splunk home location based on package, platform, and kernel
+  def self.splunk_home(platform_family, machine_kernel, package_base_name)
+    if platform_family == 'windows'
+      if machine_kernel == 'x86_64'
+        "#{ENV['PROGRAMW6432'].tr('\\', '/')}/#{package_base_name}"
+      else
+        "#{ENV['PROGRAMFILES'].tr('\\', '/')}/#{package_base_name}"
+      end
+    else
+      "/opt/#{package_base_name}"
+    end
+  end
+
+  # Returns filepath to splunk bin based on platform family
+  def self.splunk_command(node)
+    filepath = "#{node['splunk']['home']}/bin/splunk"
+
```

Comment:
I like to separate different return paths so it is just a style thing. I don't mind removing 83. If you guys feel strongly, I can remove the extra lines.